index.js

```diff
@@ -62,7 +63,11 @@ module.exports = function (p, exec, log, cwd, tasks, options) {
 			input: p.stdin,
 			output: p.stdout
 		});
-		var promptChoices = Object.keys(tasks).filter(filterPrefixes);
+		var promptChoices = Object.keys(tasks)
+			.filter(filterPrefixes)
```

Comment:
here we should convert to use an `object` with name/value instead as supported by [Inquirer.js](https://github.com/SBoudrias/Inquirer.js#question) this will allow us to print an option displaying the command and to actually run only the npm script value